internal/keepers/filter_test.go

```diff
@@ -1,6 +1,7 @@
 package keepers
 
 import (
+	"github.com/smartcontractkit/ocr2keepers/pkg/chain"
```

Comment:
I suggest to add goimports tool to your file watchers in IDE

Reply:
I do have goimports added to my file watchers, is there a specific configuration you use for it/would like me to use?

Reply:
```go
import (
    // built-in imports

    // third-party imports

    // internal imports 
)
```